pynamodb_attributes/unicode_datetime.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+from datetime import timezone
+from typing import Any
+from typing import Optional
+
+import pynamodb.attributes
+from pynamodb.attributes import Attribute
+
+
+class UnicodeDatetimeAttribute(Attribute[datetime]):
+    """
+    Stores a 'datetime.datetime' object as an ISO8601 formatted string
+
+    This is useful for wanting database readable datetime objects that also sort.
+
+    >>> class MyModel(Model):
+    >>>   created_at = UnicodeDatetimeAttribute()
+    """
+
+    attr_type = pynamodb.attributes.STRING
+
+    def __init__(
+        self,
+        force_tz: bool = True,
+        force_utc: bool = False,
+        fmt: Optional[str] = None,
+        **kwargs: Any,
+    ):
+        """
+        :param force_tz: If set it will add timezone info to the value if it does not have it
+        :param force_utc: If set it will normalize the datetime to UTC
```

Comment:
"will serialize" maybe? This doesn't affect deserializing: the deserialized values will come with whatever they were persisted with.

Reply:
Can you elaborate?

Reply:
The transformation (normalizing to UTC etc.) happens when you serialize but not when you deserialize some arbitrary thing that was already in the database.